tests/test_viewport.py

```diff
@@ -572,6 +608,33 @@ def execute(self):
         self.command("w", regex="written$", lines=1)
 
 
+class TestViewportsVisibleMetaTag(MultiSyntaxIntegrationTest):
+
+    viminput = """
+    HEADER2(Home tasks | project:Home +VISIBLE)
```

Comment:
Drop the `project:Home`, as otherwise this isn't testing the `+VISIBLE` meta tag at all, the chore task would be there anyway.